model/host/host.go

```diff
@@ -94,7 +94,6 @@ type Host struct {
 	TotalIdleTime time.Duration `bson:"total_idle_time,omitempty" json:"total_idle_time,omitempty" yaml:"total_idle_time,omitempty"`
 
 	// managed containers require different information based on host type
-	ContainerID string `bson:"container_id,omitempty" json:"container_id,omitempty"`
```

Comment:
I think the monitor code still needs this field to exist

Reply:
@john-m-liu I don't believe so -- I think we realized that containerID is the same as the container host's Id field and that this is redundant. Did you have any particular issues in mind?

Reply:
Got it. With the current implementation of the docker provider, the ID starts as a placeholder like "container-123456" so I would make sure that this gets changed/replaced when the container actually starts

Reply:
Since children keep track of their parents, the problem potentially is with the parents. If we change their ID when intent documents become true hosts. I opened https://jira.mongodb.org/browse/EVG-3501 to track.

Reply:
Good point. One approach to address this is to maintain the list of children in the parent document, and keep this container ID field (so that we don't need to change the child ID)